source/services/odnoklassniki.js

```diff
@@ -20,7 +20,7 @@ const odnoklassniki = {
     popupHeight: 400,
 };
 
-set(window, 'ODKL.updateCount', (index, counter) => {
+set(global, 'ODKL.updateCount', (index, counter) => {
```

Comment:
> it's consistent with the browser behavior

Does this bring any benefits? (I don’t see them at the moment.)

If not, I’d leave it as is because 1) `global` pollution is a real problem, and 2) I don’t know what could happen if we move `ODKL.updateCount` away from `window`.